### ISSUE TITLE
dotnet_style_require_accessibility_modifiers missing from C# Code Style settings UI

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#60237]

**Comment by andrekoehler** [comment]
I created this issue.
